gapic-generator-java/src/main/java/com/google/api/generator/gapic/composer/rest/HttpJsonServiceStubClassComposer.java

```diff
@@ -1103,6 +1151,124 @@ protected List<Expr> createOperationsStubInitExpr(
             .build());
   }
 
+  /**
+   * Build an Expr that creates an HttpRule. Creates a builder and adds the http verb, custom path,
+   * and any additional bindings.
+   *
+   * <p>`additional_bindings` can only be nested one layer deep, so we only check once
+   *
+   * @param httpRule HttpRule to be re-created for the library to send over
```

Comment:
A general tip for Javadoc, it is not [required](go/java-style#s7-javadoc) for private methods and I would suggest not to add it because it's more maintenance in the future. Comments are always welcome though.

Reply:
Ok, I'll just change this to comments. I wanted to document this since this is a one-off addition just to handle special cases for Operations.